libs/sdk-ui-tests-e2e/cypress/tools/table.ts

```diff
@@ -80,6 +82,25 @@ export class Table {
             .find(`.s-cell-${row}-${column}`);
     }
 
+    getPivotTableColumnTotalCell(row: number, column: number, match: boolean) {
```

Comment:
What about the naming, it's just a check, maybe I'm missing something but it does not return the cell. So something like hasPivotTableColumnTotalCell?

Reply:
Actually, is this method even used?

Reply:
Forgot to remove it. Thanks.